common/app/views/fragments/analytics/google.scala.html

```diff
@@ -13,9 +13,11 @@
     var pillar = '@{navMenu.currentPillar.map(_.title).getOrElse("")}';
 }
 
+// This is to prevent console errors from code calling window.ga for sessions where we have not loaded the tracking script
+// (where the user has not given their consent for GA tracking)
+window.ga = function() {};
 
-(function() {
-
+(window.initialiseGa = function() {
```

Comment:
I think you can remove the `()` wrapping this variable assign - the reason function was wrapped before was because it was an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) so I don't think this needed anymore.

Reply:
Excellent, thanks Gareth